### ISSUE TITLE
Can TsMaster do error generation?

### ISSUE THREAD

**Issue by Mkiller0420** [TOSUN-Shanghai/TSMaster#100]
Can TsMaster do error generation just like PCAN-Viewer?

![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/31810823/b00185f4-f42d-449f-a020-4c10d34f573c)

https://www.peak-system.com/produktcd/Pdf/English/PCAN-View_ErrorGenerator_eng.pdf

**Comment by freshhope** [comment]
Yes, a professional and standalone CAN/CAN FD disturbance generator is going to be released
